Build failed in Android while trying to run the example project

I did

```
git clone https://github.com/airbnb/native-navigation.git
cd native-navigation
npm install
npm start
```

Then opened another tab on the terminal and ran:
`npm run run:android`

The output was:

```
FAILURE: Build failed with an exception.

* What went wrong:
Task 'installDebug' not found in root project 'android'.

* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 55.172 secs

npm ERR! Darwin 16.4.0
npm ERR! argv "/usr/local/Cellar/node/7.2.0/bin/node" "/usr/local/bin/npm" "run" "run:android"
npm ERR! node v7.2.0
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! native-navigation@0.0.5 run:android: `cd example/android && ./gradlew installDebug && cd - && npm run start:android`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the native-navigation@0.0.5 run:android script 'cd example/android && ./gradlew installDebug && cd - && npm run start:android'.
```

I'm on macOS Sierra 10.12.3, Node 7.2.0

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "run:packager": "./node_modules/react-native/packager/packager.sh",
     "run:ios": "react-native run-ios --project-path ./example/ios",
     "start:android": "adb shell am start -n com.airbnb.android.react.navigation.example/.MainActivity",
-    "run:android": "cd example/android && ./gradlew installDebug && cd - && npm run start:android",
+    "run:android": "./gradlew installDebug && npm run start:android",
     "lint": "eslint ./",
     "build": "npm run build:js && npm run build:android && npm run build:ios",
     "build:js": "exit 0",

